docs/guide/replacement-guides/rimraf.md

```diff
@@ -0,0 +1,44 @@
+# Replacements for `rimraf`
+
+## Node.js
+
+Node.js v14.14.0 and above provide a native alternative: [`fs.rm`](https://nodejs.org/api/fs.html#fspromisesrmpath-options). It supports recursive deletion and works as a direct replacement.
+
+```js
+import { rm } from 'node:fs/promises' // [!code ++]
+import rimraf from 'rimraf' // [!code --]
+
+await rimraf('./dist') // [!code --]
+await rm('./dist', { recursive: true, force: true }) // [!code ++]
+```
+
+## Node.js (before v14.14.0)
+
+If you need to support Node.js 12 up to 14.13, you can use [`fs.rmdir`](https://nodejs.org/api/fs.html#fsrmdirpath-options-callback) with the recursive option. This was added in Node v12.10.0, though it’s deprecated as of Node v14.
+
+```js
+import { rmdir } from 'node:fs/promises'
+import rimraf from 'rimraf' // [!code --]
+
+await rimraf('./dist') // [!code --]
+await rmdir('./dist', { recursive: true }) // [!code ++]
+```
+
+## CLI usage
+
+To replace `rimraf` inside npm scripts, you can run Node directly in eval mode:
+
+```sh
+node -e "require('fs').rmSync('./dist', { recursive: true, force: true, maxRetries: process.platform === 'win32' ? 10 : 0 })"
+```
+
+---
+
+## `premove`
+
+If you are on an older Node.js version (before v12.10) or you specifically need a CLI replacement, use [`premove`](https://github.com/lukeed/premove). It provides both an API and a CLI and works on Node.js v8 and newer.
```

Comment:
> If you are on an older Node.js version (before v12.10) or you specifically need a CLI replacement, you can use [...]

instead of saying `use premove`, `you can use premove` (since this whole doc is a list of suggested replacements rather than us telling you to use one or the other)